.changeset/big-parents-sleep.md

```diff
@@ -0,0 +1,220 @@
+- Updated `color.decorative.blue` from `base.color.blue.700` to `base.color.blue.900updated`.
+- Updated `color.decorative.neutral` from `base.color.grey.700` to `base.color.grey.650`.
+- Updated `color.datavis.categorical-10` from `#7372CF` to `base.color.blue.200`.
+- Updated `color.datavis.categorical-20` from `base.color.datavis.gold2` to `base.color.grenn.400`.
+  Updated `color.datavis.categorical-30` from `base.color.datavis.pink2` to `base.color.purple.200`.
+- Updated `color.datavis.categorical-40` from `base.color.datavis.lightblue1` to `base.color.blue.500`.
+- Updated `color.datavis.categorical-50` from `#A68A74` to `base.color.fuschia.300`.
+- Updated `color.datavis.categorical-60` from `#B889FF` to `base.color.cyan.300`.
+- Updated `color.datavis.categorical-70` from `#4D8DA8` to `base.color.plum.100`.
+- Updated `color.datavis.categorical-80` from `base.color.datavis.purple3` to `base.color.coral.200`.
+- Deprecated `color.decorative.teal`, use `color.decorative.cyan` instead.
+- Deprecated `color.decorative.red`.
+- Deprecated `color.decorative.orange`.
+- Deprecated `color.decorative.yellow`.
+
+**Components**
+
+- `button.primary.rest.background` updated from `color.background.primary.strong` to `color.background.neutral.xstrong`.
+- `button.primary.rest.textColor` updated from `color.text.onPrimaryStrong` to `color.text.onStrong`.
+- `button.primary.rest.iconColor` updated from `color.icon.onPrimaryStrong` to `color.icon.onStrong`.
+- `button.primary.rest.fontWeight` updated from `fontWeight.semibold` to `fontWeight.medium`.
+  - Referenced by `button.primary.hover.fontWeight`, `button.primary.disabled.rest.fontWeight`, `button.primary.selected.rest.fontWeight`, `button.primary.selected.hover.fontWeight`
+- `button.primary.hover.background` updated from `color.background.primary.strong.hover` to `color.background.neutral.strong`.
+- `button.primary.hover.textColor` updated from `color.text.onPrimaryStrong` to `color.text.onStrong`.
+- `button.primary.hover.iconColor` updated from `color.icon.onPrimaryStrong` to `color.icon.onStrong`.
+- `button.primary.selected.rest.background` updated from `color.background.primary.strong` to `color.background.neutral.xstrong`.
+- `button.primary.selected.rest.textColor` updated from `color.text.onPrimaryStrong` to `color.text.onStrong`.
+- `button.primary.selected.rest.iconColor` updated from `color.icon.onPrimaryStrong` to `color.icon.onStrong`.
+- `button.primary.selected.hover.background` updated from `color.background.primary.strong.hover` to `color.background.neutral.strong`.
+- `button.primary.selected.hover.textColor` updated from `color.text.onPrimaryStrong` to `color.text.onStrong`.
+- `button.primary.selected.hover.iconColor` updated from `color.icon.onPrimaryStrong` to `color.icon.onStrong`.
+
+- `button.secondary.rest.textColor` updated from `color.text.primary` to `color.text.strong`.
+  - Referenced by `button.secondary.hover.textColor`
```

Comment:
Were these previously referencing the rest textColor as well? Why referencing rest vs. pointing directly to `color.text.strong`? Easier to keep synched?